lib/mutant/mutator/node/literal/regex.rb

```diff
@@ -63,6 +63,8 @@ def body_expression
           #
           # @return [Array<Parser::AST::Node>]
           def body
+            # TODO: also: kill 0...1 mutation -- how the heck does this happen?
+            # manually inserting it causes unit tests to fail.
             children.slice(0...-1)
```

Comment:
@mbj This is the mysterious mutation that fails the unit tests if you actually insert it:

```diff
def body
-  children.slice(0...-1)
+  children.slice(0...1)
 end
 ```

Reply:
Must be a test selection issue. Very likely the examples from the meta are tagged `:regexp` and mutants test selection overwrite does not catch the various regexp subtypes. 

I can look into providing a better selection hint.

Reply:
@dgollahon Is that still a problem?

Reply:
Yeah, last I checked. If you have any suggestions here i'd appreciate it.

Reply:
Oh, and another thing with this method @mbj: It fails with multiple mutations on 2.7 but only one on 2.6. I believe that's because of beginless ranges in 2.7. What's our policy for working around that? I could lift the range to a constant or add an `--ignore-subject`. That would also resolve this other mutation but I'm not sure if that's what you'd like me to do.

Also, I was wondering: is there a reason we don't check mutation coverage on 2.5?

Reply:
@dgollahon Lets go with ignore subject for the moment.

I'll produce a version target soon.